### ISSUE TITLE
NOTIS editing for codabar barcode?

### ISSUE THREAD

**Issue by Jack-Gitter** [metafloor/bwip-js#252]
Is it possible to enable/disable NOTIS editing for the Codabar barcode type? The scanners that I'm dealing with are programmed to scan Codabar with this mode enabled, so I want to figure out a way to strip the start/stop characters from the barcode if possible.

**Comment by metafloor** [comment]
Have you looked at the BWIPP reference for codabar?  bwip-js is essentially a wrapper around BWIPP.

**Comment by Jack-Gitter** [comment]
> Have you looked at the BWIPP reference for codabar? bwip-js is essentially a wrapper around BWIPP.

Yes, I took a look at the codabar page here, https://github.com/bwipp/postscriptbarcode/wiki/Codabar but unfortunately it doesn't say anything about NOTIS mode, and was wondering if anyone here had some extra information

**Comment by terryburton** [comment]
> Is it possible to enable/disable NOTIS editing for the Codabar barcode type? The scanners that I'm dealing with are programmed to scan Codabar with this mode enabled, so I want to figure out a way to strip the start/stop characters from the barcode if possible.

That's confused.

"NOTIS editing" refers to stripping the AB start/stop characters from a symbol at scan time, not during generation. If you stripped these characters from the symbol itself then you would no longer have a Codabar symbol, by definition.
